workspace-loader/src/workspace-loader.ts

```diff
@@ -208,12 +208,17 @@ export class WorkspaceLoader {
             masterApi.subscribeInstallerOutput(this.workspace.id,
                  (message: any) => this.onEnvironmentOutput(message.text));
             masterApi.subscribeWorkspaceStatus(this.workspace.id,
-                (message: any) => {
+                (message: WorkspaceStatusChangedEvent) => {
                     if (message.error) {
                         reject(new Error(`Failed to run the workspace: "${message.error}"`));
                     } else if (message.status === 'RUNNING') {
                         this.checkWorkspaceRuntime().then(resolve, reject);
                     } else if (message.status === 'STOPPED') {
+                        if (message.prevStatus == 'STARTING') {
+                            this.loader.error('Workspace unexpected stopped during starting phase.');
```

Comment:
I think "Workspace unexpected stopped" is enough.
WDYT?

Reply:
I simplified error message.